packages/nextjs/components/scaffold-eth/Balance.tsx

```diff
@@ -44,6 +55,8 @@ export const Balance = ({ address, className = "", usdMode }: BalanceProps) => {
     );
   }
 
+  const formattedBalance = Number(balance?.formatted);
```

Comment:
Number(undefined) is `NaN`, so it will break further if there is no balance. Better to check if balance exists `balance ? Number(balance.formatted) : 0`

Reply:
good catch! 
commit with the suggested change: [9bbc4c2f5386d9385fb404bf86b031636fa5bfa6](https://github.com/scaffold-eth/scaffold-eth-2/pull/788/commits/9bbc4c2f5386d9385fb404bf86b031636fa5bfa6)
@technophile-04 s/o to you for also recognizing the error!